tests/lightning_qubit/test_measurements_class.py

```diff
@@ -419,6 +424,7 @@ def calculate_reference(tape, lightning_sv):
         return m.measure_final_state(tape)
 
     @flaky(max_runs=5)
+    @pytest.mark.parametrize("shots", [None, 1000000, (900000, 900000)])
```

Comment:
Do we need 1M shots here?

Reply:
You are right, I can reduce one order of magnitude the number of shots (100k), less than that, the test fails :rocket: